lib/Service/OpenAiSettingsService.php

```diff
@@ -334,7 +342,7 @@ public function setQuotas(array $quotas): void {
 			}
 		}
 
-		$this->config->setAppValue(Application::APP_ID, 'quotas', json_encode($quotas));
+		$this->appConfig->setValueString(Application::APP_ID, 'quotas', json_encode($quotas));
```

Comment:
we can add a flag for `json_encode` to throw here or should do a `false` check, It is fine in other places but this might return false.

Reply:
:+1: